test/functional/feature_llmq_connections.py

```
@@ -110,18 +113,18 @@
     def check_reconnects(self, expected_connection_count):
         self.log.info("disable and re-enable networking on all masternodes")
-        for mn in self.mninfo:
-            mn.node.setnetworkactive(False)
-        for mn in self.mninfo:
-            self.wait_until(lambda: len(mn.node.getpeerinfo()) == 0)
-        for mn in self.mninfo:
-            mn.node.setnetworkactive(True)
+        for mn in self.mninfo: # type: MasternodeInfo
+            mn.get_node(self).setnetworkactive(False)
+        for mn in self.mninfo: # type: MasternodeInfo
+            self.wait_until(lambda: len(mn.get_node(self).getpeerinfo()) == 0)
```

Ruff (0.11.9)

Function definition does not bind loop variable `mn`

(B023)